py-polars/polars/eager/series.py

```diff
@@ -114,13 +114,6 @@ def wrap_s(s: "PySeries") -> "Series":
     return Series._from_pyseries(s)
 
 
-def _maybe_cast(el: "Type[DataType]", dtype: Type) -> "Type[DataType]":
```

Comment:
It feels wrong to import a private variable into another module, so I moved this function to `datatypes.py`.